src/xo-app/pool/tab-logs.js

```diff
@@ -1,111 +1,59 @@
+    sortCriteria: 'body'
+  }
+]
 
-    this.state = {
-      page: 1
-    }
+const INDIVIDUAL_ACTIONS = [
+  {
+    handler: deleteMessage,
+    icon: 'delete',
+    label: 'deleteLog'
   }
+]
 
-  _deleteAllLogs = () => map(this.props.logs, deleteMessage)
-  _nextPage = () => this.setState({ page: Math.min(this.state.page + 1, this.getNPages()) })
-  _previousPage = () => this.setState({ page: Math.max(this.state.page - 1, 1) })
+export default class TabLogs extends Component {
+  _deleteAllLogs = logs => Promise.all(map(logs, deleteMessage))
 
   render () {
-    const logs = this.getLogs()
-    const { page } = this.state
-
-    return <Container>
-      {isEmpty(logs)
-        ? <Row>
-          <Col mediumSize={6} className='text-xs-center'>
-            <br />
-            <h4>{_('noLogs')}</h4>
-          </Col>
-        </Row>
-        : <div>
-          <Row>
-            <Col className='text-xs-right'>
-              <TabButton
-                btnStyle='secondary'
-                disabled={page === 1}
-                handler={this._previousPage}
-                icon='previous'
-              />
-              <TabButton
-                btnStyle='secondary'
-                disabled={page === this.getNPages()}
-                handler={this._nextPage}
-                icon='next'
-              />
-              <TabButton
-                btnStyle='danger'
-                handler={this._removeAllLogs} // FIXME: define this method
-                icon='delete'
-                labelId='logRemoveAll'
-              />
-            </Col>
-          </Row>
-          <Row>
-            <Col>
-              <table className='table'>
-                <thead className='thead-default'>
-                  <tr>
-                    <th>{_('logDate')}</th>
-                    <th>{_('logName')}</th>
-                    <th>{_('logContent')}</th>
-                    <th>{_('logAction')}</th>
-                  </tr>
-                </thead>
-                <tbody>
-                  {map(logs, log =>
-                    <tr key={log.id}>
-                      <td><FormattedTime value={log.time * 1000} minute='numeric' hour='numeric' day='numeric' month='long' year='numeric' /> (<FormattedRelative value={log.time * 1000} />)</td>
-                      <td>{log.name}</td>
-                      <td>{log.body}</td>
-                      <td>
-                        <ActionRow
-                          btnStyle='danger'
-                          handler={deleteMessage}
-                          handlerParam={log}
-                          icon='delete'
-                        />
-                      </td>
-                    </tr>
-                  )}
-                </tbody>
-              </table>
-            </Col>
-          </Row>
-        </div>
+    const GROUPED_ACTIONS = [
+      {
+        handler: this._deleteAllLogs,
+        icon: 'delete',
+        label: 'deleteLogs'
       }
-    </Container>
+    ]
```

Comment:
You don't need to create a new array on each render. Create it ouside the class and inline the `_deleteAllLogs`.